libs/deer/README.md

```diff
@@ -4,6 +4,10 @@
 
 `deer` is an **experimental** backend-agnostic deserialization framework for Rust, featuring meaningful error messages and context (utilizing [`error-stack`](https://crates.io/crates/error-stack)) and a fail-slow behavior by default.
 
+## ⚠️ Disclaimer ⚠️
+
+This crate does not ship with any functionality and is only a name reservation to stop potential name squatting. In the future the actual crate will be published under this name, for the current (incomplete) implementation of the crate please visit the linked repository.
+
```

Comment:
I can add this section when publishing the crates, we don't need it in the repository I guess
```suggestion
```